soda/core/soda/execution/data_source.py

```diff
@@ -499,7 +499,9 @@ def histogram_sql_and_boundaries(
 
         if min_value >= max_value:
             self.logs.warning(
-                f"Min of {column_name} on table: {table_name} must be smaller than max value. Min is {minimum}, and max is {maximum}"
+                "Min of {column_name} on table: {table_name} must be smaller than max value. Min is {min_value}, and max is {max_value}".format(
+                    column_name=column_name, table_name=table_name, min_value=min_value, max_value=max_value
+                )
```

Comment:
Can you remove this since it won't be part of this PR?

Reply:
done by merging the other PR